queries/js/lightInfo.js

```diff
@@ -0,0 +1,82 @@
+"use strict";
+
+const XMLHttpRequest = require("xmlhttprequest").XMLHttpRequest;
+
+function lightInfo(ipAddress, username) {
+
+    var request = new XMLHttpRequest();
+    request.open('GET', `http://${ipAddress}/api/${username}/lights`, false);  // `false` makes the request synchronous
+    request.send(null);
+
+    if (request.status === 200) {
+        return JSON.parse(request.responseText);
+    }
+    else {
+        throw new Error('Error while fetchin light info');
```

Comment:
*fetching

Reply:
Yeah, this branch needs a lot of work to meet our standards :). Will get on it tomorrow.